vrs/marten-construction-script.py

```diff
@@ -1,168 +1,180 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-# Daniel Marten 01-04-2022
-# Constructing a ~100k Variant Test Database for GA4GH-VRS Annotation Script
-# NOTE: adapted from a Jupyter Notebook, so excuse any formatting quirks! 
-# NOTE: please make sure to change the paths for checkpointing!!!
-# Edits on: 01-06-2022
+# Daniel Marten 01-20-2023
+# Construct a ~100k Variant Test VCF for GA4GH-VRS Project
+# Reach out to me at: marten@broadinstitute.org
 
+# Import and set up important packages/modules
 import hail as hl
 import random
 from random import sample
 from gnomad.resources.grch38.gnomad import public_release
 
-# Setting randomness
+from gnomad.resources.resource_utils import (
+    MatrixTableResource,
+    VariantDatasetResource,
+    VersionedMatrixTableResource,
+    VersionedVariantDatasetResource,
+)
+
+# Import logging for logs (to replace print statements)
+import logging
```

Comment:
echoing Kristen's sentiments here re: `isort`. the import order should be something like this:
```
import logging
import random

import hail as hl

from gnomad.resources.resource_utils import (
    MatrixTableResource,
    VariantDatasetResource,
    VersionedMatrixTableResource,
    VersionedVariantDatasetResource,
)
```